base/system/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@
 add_subdirectory(autochk)
 add_subdirectory(bootok)
 add_subdirectory(chkdsk)
+add_subdirectory(conime)
```

Comment:
Wouldn't it be better to place it in win32ss/user ? Since it is for the win32 console that also resides there.

Reply:
Moved.